src/scss/_sofie-utility-classes.scss

```diff
@@ -270,6 +270,14 @@
   visibility: hidden !important;
 }
 
+.relative {
+  position: relative !important;
+}
+
+.absolute {
+  position: absolute !important;
+}
```

Comment:
Are you using this class?
We should try to refrain from adding code that we are not using. We can always add it once we actually need it.

Reply:
Removed